vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp

```diff
@@ -42,6 +42,28 @@ function drawLinks() {
         $("#space").append(link);
     }
     </c:forEach>
+    
+    <c:forEach items="${externalLinkList}" var="link">
+    {
+        var posX = parseInt($("#space").css('margin-left')) + $("#space").position().left; 
+        var posY = $("#space").position().top;
+        var link = $('<a></a>');
+        link.attr('href', '<c:url value="http://${link.externalLink.externalLink}"/>');
+        
+        var linkDisplay = $('<span data-feather="navigation-2" class="flex"></span>');
```

Comment:
Let's use the font awesome globe icon instead of feature icons:
fas fa-globe-americas

Reply:
?